spec/system/support_interface/providers_spec.rb

```diff
@@ -42,6 +42,9 @@ def when_i_visit_the_providers_page
   def and_providers_are_configured_to_be_synced
     allow(Rails.application.config).to receive(:providers_to_sync)
```

Comment:
Do we need a card to remove the YAML etc?

Reply:
Was just chatting with Steve about this. It sounds like we only currently need this for the local test data generation, so it can remain for now, but we may need a different mechanism to customise sandbox environments, not through a yaml file but through something more pipeline specific.

Reply:
Yes it's still used for the `setup_local_dev_data` rake task. We could have a card to come up with a better way to do that, maybe that would be covered by the sandbox work?